ansible/roles/test/tasks/pfc_wd/functional_test/functional_test.yml

```diff
@@ -0,0 +1,344 @@
+- conn_graph_facts: host={{ peer_device }}
+  connection: local
+
+- name: Prepare variables required for PFC test
+  set_fact:
+    pfc_queue_index: 4
+    pfc_frames_number: 100000
+    pfc_fanout_interface: "{{neighbors[pfc_wd_test_port]['peerport']}}"
+    peer_hwsku: "{{device_info['HwSku']}}"
+    testname: functional_test
+    config_files:
+      - pfc_wd_drp_action.json
+      - pfc_wd_fwd_action.json
+      - pfc_wd_timer.json
+      - pfc_wd_del_action.json
+
+- set_fact: pfc_wd_test_neighbor_addr="{{ pfc_wd_test_port_addr | regex_replace('(^.*\.).*$', '\\1') }}{{pfc_wd_test_port_addr.split('.')[3] | int + 1 }}"
```

Comment:
These information can also be obtained from minigraph facts, minigraph_bgp.

Reply:
Done